charts/incubator/hyperswitch-app/templates/misc/configmap.yaml

```diff
@@ -33,9 +33,52 @@ data:
     enabled = true
```

Comment:
Are we using this file anywhere? If so, can we do `{{ .Files.Get "misc.toml" | nindent 4 }}` here? If not, can we remove this?

Reply:
We can remove this, we are not using it anywhere